e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+describe('System Console - Subscriptions section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        gotoSubscriptionScreen();
+    });
+
+    beforeEach(() => {
+        // # Click on close button of subscribe window if exist
+        cy.get('body').then(($body) => {
+            if ($body.find('#closeIcon').length > 0) {
+                cy.get('#closeIcon').parent().click();
+            }
+        });
+    });
+
+    it('MM-T4118 - Visibility of Trail tag on Subscription screen', () => {
+        // * Check for visibility of Trail tag
+        cy.contains('span', 'Trial', {timeout: 10000}).should('be.visible');
+    });
+
+    it('MM-T4119 - User Count on Subscription screen', () => {
+        // * Check for User count
+        cy.request('/api/v4/analytics/old?name=standard&team_id=').then((response) => {
+            cy.get('.PlanDetails__userCount > span').invoke('text').then((text) => {
+                const userCount = response.body.find((obj) => obj.name === 'unique_user_count');
+                expect(text).to.contain(userCount.value);
+            });
+        });
+    });
+
+    it('MM-T4126 - Contact Sales navigation from Subscription screen', () => {
+        // * Check for Contact Sales navigation
+        cy.window().
+            its('store').
+            invoke('getState').
+            then((state) => {
+                const currentiuserid = state.entities.users.currentUserId;
+                const email = encodeURIComponent(state.entities.users.profiles[currentiuserid].email);
+                const name = encodeURIComponent(`${state.entities.users.profiles[currentiuserid].first_name} ${state.entities.users.profiles[currentiuserid].last_name}`);
```

Comment:
I'm afraid this is webapp implementation details. I'd suggest to get the admin user via API, like
```javascript
    let adminUser;

    before(() => {
        // ...
        cy.apiGetMe().then(({user}) => {
            adminUser = user;
        });
        // ...
    });
```

Then,
```javascript
const email = encodeURIComponent(adminUser.email);
const fullName = encodeURIComponent(`${adminUser.first_name} ${adminUser.last_name}`);

cy.wrap({href: getHref}).invoke('href').should('contains', `${Cypress.env('cwsUrl')}/cloud/contact-us?email=${email}&name=${fullName}&inquiry=sales&inquiry-issue=trial_questions`);
```

Same can be applied to `it` blocks.

Reply:
Updated code as per feedback comment.